code/modules/atmospherics/machinery/components/unary_devices/machine_connector.dm

```diff
@@ -0,0 +1,111 @@
+///To be used when there is the need of an atmos connection without repathing everything (eg: cryo.dm)
+/obj/machinery/atmospherics/components/unary/machine_connector
+	name = "machine connector"
+	desc = "Internal connector to interface with the pipes."
+
+	can_unwrench = FALSE
+
+	use_power = NO_POWER_USE
+	layer = GAS_FILTER_LAYER
+	hide = TRUE
+	shift_underlay_only = FALSE
+
+	pipe_flags = PIPING_ONE_PER_TURF
```

Comment:
what happens if you try and wrench down a cryotube on a turf that already has a pipe on that layer?

Reply:
i'll have to try that